### DISCUSSION TITLE
High Availability

### DISCUSSION THREAD

**Discussion by r-moore** [quirrel-dev/quirrel#1020] [Q&A] [answered]
Does quirrel have a suggested way of deploying with HA?

i.e. multiple instances running behind a load balancer, but with a single virtual queue that survives a single node failure.

If so, what would the configuration steps look like?

**Comment by Skn0tt** [comment, marked as answer]
I'm not aware of an existing HA setup with Quirrel, so take the following with a grain of salt:

Quirrel is built in a stateless way. You can run multiple instances behind a load balancer, and point them to a HA Redis instance. Keep in mind that Quirrel is *not* compatible with Redis Clusters, though: https://github.com/quirrel-dev/owl#compatibility-with-redis-cluster
